WcaOnRails/app/webpacker/components/CompetitionOverview/CompetitionTable.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+
+import I18n from '../../lib/i18n';
+
+function calculateDate(comp, mode) {
```

Comment:
A better name might be `calculateDayDifference` or `getDayDifference` or something, since this isn't returning a date.

Reply:
good point